core/src/banking_stage.rs

```diff
@@ -95,6 +96,7 @@ pub struct BankingStageStats {
     current_buffered_packet_batches_count: AtomicUsize,
     rebuffered_packets_count: AtomicUsize,
     consumed_buffered_packets_count: AtomicUsize,
+    packet_batch_indices_len: Histogram,
```

Comment:
indexes vs indices? :)

Reply:
"indexes" would be consistent with the name of the thing that its measuring.  then again, already cleared CI, so...  up to you!

Reply:
ok game time decision, going to switch it up real quick